Plugins/NativeEngine/Source/NativeEngine.h

```diff
@@ -43,11 +43,26 @@ namespace Babylon
     struct ProgramData final
     {
         ProgramData() = default;
-        ProgramData(ProgramData&& other) = delete;
         ProgramData(const ProgramData&) = delete;
-        ProgramData& operator=(ProgramData&& other) = delete;
         ProgramData& operator=(const ProgramData& other) = delete;
```

Comment:
Just to be consistent

```suggestion
        ProgramData& operator=(const ProgramData&) = delete;
```